### ISSUE TITLE
[build] LESS 4 is not supported by gulp-less->accord dependency

### ISSUE THREAD

**Issue by goodwin74** [fomantic/Fomantic-UI#1893]
# Bug Report
Error less in build to laravel 8 (phpstorm ide)

## Steps to reproduce
1. npm install fomantic-ui
2. cd node_modules/fomantic-ui
3. gulp install

## Expected result
setup gulp

## Actual result
Error: less version 4.1.1 is not currently supported

## Version
last

The error is not clear why he swears at less which is in the root of the project, if the fomantic-ui folder has its own npm environment. There is less 3.7.0 in the modules inside the fomantic-ui folder.
And in the root of the project in node_modules less 4.1.1
How to be? How to build fomantic and why is it throwing an error?

**Comment by lubber-de** [comment]
Possible duplicate of #1818 
LESS 4.x support has been fixed for 2.8.8 (released shortly) by #1819 

Try the nightly 2.8.8 beta build 

```
npm install fomantic-ui@nightly
```

and install by `npx gulp build` inside the fomantic-ui folder

**Comment by goodwin74** [comment]
> Possible duplicate of #1818
> LESS 4.x support has been fixed for 2.8.8 (released shortly) by #1819
> 
> Try the nightly 2.8.8 beta build
> 
> ```
> npm install fomantic-ui@nightly
> ```
> 
> and install by `npx gulp build` inside the fomantic-ui folder

not work. Print error 
![noname](https://user-images.githubusercontent.com/15101984/108760131-50542700-755e-11eb-93a4-2b92600cf6bb.png)

**Comment by lubber-de** [comment]
uhm, yes, maybe because the package.json still wants 3.7... you may try to adjust the line in package.json

```
"less": "^3.7.0",
```

by

```
"less": "^4.0.0",
```

**Comment by lubber-de** [comment]
I added a PR to support existing less 4.1.1 by #1894 
run `npm update` after the adjustments in package.json

**Comment by lubber-de** [comment]
however gulp-less still wants less 3.7 so unless gulp-less gets an update you may need to adjust the package.json manually or stick to less 3.x

**Comment by lubber-de** [comment]
ok, my PR does not help, reason is the latest version of gulp-less 4.0.1 does not support less 4, so we are stuck at less 3.x until then.
If you only need the default theme, you may try to use https://github.com/fomantic/Fomantic-UI-LESS instead (or without compiling at all the precompiled CSS package https://github.com/fomantic/Fomantic-UI-CSS

**Comment by lubber-de** [comment]
There already exists an issue request to support less 4 in gulp-less by https://github.com/gulp-community/gulp-less/issues/312

**Comment by goodwin74** [comment]
> ok, my PR does not help, reason is the latest version of gulp-less 4.0.1 does not support less 4, so we are stuck at less 3.x until then.
> If you only need the default theme, you may try to use https://github.com/fomantic/Fomantic-UI-LESS instead (or without compiling at all the precompiled CSS package https://github.com/fomantic/Fomantic-UI-CSS

Can I use the less version (fomantic-ui-less) to build CSS and JS, just the components I need? If so, how can this be done?
I need two CSS and JS files for Laravel that I can put in Mix

**Comment by goodwin74** [comment]
> however gulp-less still wants less 3.7 so unless gulp-less gets an update you may need to adjust the package.json manually or stick to less 3.x

Did I understand correctly that I need to install the fomantic-ui beta with the "ignore_scripts" attribute, change the package.json inside the fomantic-ui folder, replace the value 3.7.0 with 4.1.1.
And do npx gulp build?

**Comment by lubber-de** [comment]
> Did I understand correctly that I need to install the fomantic-ui beta with the "ignore_scripts" attribute, change the package.json inside the fomantic-ui folder, replace the value 3.7.0 with 4.1.1.
> And do npx gulp build?

As this will still use gulp-less (i believe this will still trigger the error

> Can I use the less version (fomantic-ui-less) to build CSS and JS, just the components I need? If so, how can this be done?
I need two CSS and JS files for Laravel that I can put in Mix

The LESS Repo is basically meant to be used inside other LESS related project (where you would only include the semantic.less file)

If you really only need some CSS/JS files with the default theme, then i suggest to fetch them precompiled either from the dist folder of this repo or the precompiled [CSS Repo](https://github.com/fomantic/Fomantic-UI-CSS).

This way you dont need to hassle with building/less at all

**Comment by goodwin74** [comment]
> > Did I understand correctly that I need to install the fomantic-ui beta with the "ignore_scripts" attribute, change the package.json inside the fomantic-ui folder, replace the value 3.7.0 with 4.1.1.
> > And do npx gulp build?
> 
> As this will still use gulp-less (i believe this will still trigger the error
> 
> > Can I use the less version (fomantic-ui-less) to build CSS and JS, just the components I need? If so, how can this be done?
> > I need two CSS and JS files for Laravel that I can put in Mix
> 
> The LESS Repo is basically meant to be used inside other LESS related projects (where you would only include the semantic.less file)
> 
> If you really only need some CSS/JS files with the default theme, then i suggest to fetch them precompiled either from the dist folder of this repo or the precompiled [CSS Repo](https://github.com/fomantic/Fomantic-UI-CSS).
> 
> This way you dont need to hassle with building/less at all
> 
> Also available via CDN
> https://cdn.jsdelivr.net/npm/fomantic-ui@nightly/dist/components/YOUR_DESIRED_FILE.css/.js

If I install Less version 3.7.0, can I bundle a CSS and JS framework with only the components I need into two files?
I would not like to write a large number of CSS and JS files in laravel mix.
It is convenient when in laravel mix I specified two files (js and css) in dist, and in the fomantic-ui configuration I commented out unnecessary components and launched the build.

**Comment by lubber-de** [comment]
LESS 4 will work once the following PR is merged into gulp-less because the accord repo which gulp-less uses is dead
https://github.com/gulp-community/gulp-less/pull/313
